theme_buzzy/views/snippets/s_image_title.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image_title" inherit_id="website.s_image_title">
+   <xpath expr="//section" position="attributes">
+      <attribute name="class" add="o_cc4" remove="o_cc5 oe_img_bg" separator=" "/>
+      <attribute name="style"/>
+      <attribute name="data-oe-shape-data">{'shape':'web_editor/Blobs/04','flip':[],'showOnMobile':false,'shapeAnimationSpeed':'0'}</attribute>
+   </xpath>
+   <xpath expr="//div[hasclass('container')]" position="before">
+      <div class="o_we_shape o_web_editor_Blobs_04"/>
+   </xpath>
+   <xpath expr="//div[hasclass('o_we_bg_filter')]" position="replace"/>
```

Comment:
Incorrect space indent. Should be 4, not 3

Reply:
What an eye ! 👁️ Nice catch !
I updated the branch, the diff : https://github.com/odoo/design-themes/compare/4811d775b9fcfc8cc6d4824d643f54d824cff18c..6b24a8594f451c41cc0538a55c4ee44df86f1f60